config/krakend/krakend.json

```diff
@@ -32,7 +32,7 @@
     {{ template "workflow_proxy.tmpl" .workflow_proxy_settings.endpoints }},
     {{ template "workflow_proxy.tmpl" .workflow_proxy_settings_deprecated.endpoints }},
     {{ template "uniconfig.tmpl" .uniconfig_settings }},
-    {{ template "uniconfig_deprecated.tmpl" .uniconfig_settings_deprecated }},
```

Comment:
have the deprecated settings been removed ?

Reply:
That's why I am asking ... whether this file should be removed from the codebase:

./config/krakend/settings/uniconfig_settings_deprecated.json